examples/hosts/app-integration/external-controller/src/app.ts

```diff
@@ -49,9 +47,10 @@ async function start(): Promise<void> {
     // Create a custom ITelemetryBaseLogger object to pass into the Tinylicious container
     // and hook to the Telemetry system
     const consoleLogger: ConsoleLogger = new ConsoleLogger();
+    const tinyliciousClient = new TinyliciousClient();
 
     // Get or create the document depending if we are running through the create new flow
-    const client = useFrs ? FrsClient : TinyliciousClient;
+    const client = useFrs ? FrsClient : tinyliciousClient;
```

Comment:
better to create it on this line instead of above imo

Reply:
+1 to above. Otherwise tinyliciousClient always gets created

Reply:
right, I was actually debating the same thing when refactoring, but chose to do it this way for demonstration/readability purpose. Efficiency is definitely a bigger concern so will change it.